js/modules/validators.js

```diff
@@ -0,0 +1,26 @@
+
+export const emailValidator = (email) => {
+    const pattern = /^(.+)([a-z]+)([0-9]+)@(gmail)\.(com)\s*$/i;
```

Comment:
What happens when your email is not GMAIL?

Reply:
![image](https://user-images.githubusercontent.com/51375428/149081061-034297bd-1dca-4e52-ae67-090f0ba90f22.png)

Your validation is not working.